src/createStore.ts

```diff
@@ -76,21 +75,23 @@ export function createStore<
 export function createStore<
   S,
   A extends Action,
+  PreloadedState,
```

Comment:
Why not add this as the last param and make it have a default? That would be less breaking.

Reply:
I'm open to it. It feels better to keep the generics for the reducer next to each other and then have the generics for the enhancer next to each other. Considering that we don't encourage direct use of `createStore` or manually specifying generics, it feels worth it to me, but I'm open to whatever you, @markerikson, and @phryneas think is best.

Reply:
While *hopefully*, nobody would ever write those out, but have them inferred by parameter usage, I agree - new arguments should be added at the end, not somewhere in the middle.